docker/README.md

```diff
@@ -0,0 +1,35 @@
+#  How to run the multi-validateer docker setup
```

Comment:
Beautiful readme.

Reply:
Courtesy of @gaudenzkessler in most parts 😄